tox.ini

```diff
@@ -94,7 +94,7 @@ commands =
 description = Run integration tests
 deps =
     aiohttp
-    juju
+    juju~=3.0.0
```

Comment:
Should this maybe be `juju~=3.0`?

Reply:
`juju<3.1` would also work. All 3 options are equivalent afaict.

Reply:
They aren't _quite_ equivalent, because there is actually `3.0.4`. `< 3.1` is clearly enough "the highest possible 3.0 release"